cwlavro-tools/src/main/java/io/cwl/avro/CWL.java

```diff
@@ -103,6 +105,48 @@ private String convertCWLType(Object cwlType) {
         return type;
     }
 
+    private static Object getStubForItem(String strType, Object stub, String value) {
```

Comment:
I need the default value for stub to be returned in case it doesn't match any switch cases.